spec/controllers/avail_links_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Preferences::AvailLinksController, type: :controller do
+  setup :activate_authlogic
+  before do
+    @account = Account.create! username: 'testaccount', password: '123456', password_confirmation: '123456'
+    @plan = Plan.create! account: @account, edit_text: ''
+    @account_session = AccountSession.create! @account
+  end
+
+  before do
+    @avail_links = [
+      FactoryGirl.create(:avail_link),
+      FactoryGirl.create(:avail_link),
+      FactoryGirl.create(:avail_link),
+    ]
+
+    FactoryGirl.create :opt_link, account: @account, avail_link: @avail_links[0]
+    FactoryGirl.create :opt_link, account: @account, avail_link: @avail_links[2]
+  end
+
+  describe '#show' do
+    before { get :show }
+    subject { response }
+    it { is_expected.to be_success }
+    it { is_expected.to render_template('show') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.